scripts/trigger_oozie_job.sh

```diff
@@ -35,11 +37,14 @@ echo "Successfully transfered ./configuration/${ENV}/job.properties to bi-${ENV}
 
 # Trigger the oozie job and get the job id, remove unused chars from the id and then poll it.
 # JOB_ID is something like 'job: 213871982-213123123-asdasd', we remove 'job: '
-ssh -tt bi-${ENV}-ci@${HOST} OOZIE_HOME=$OOZIE_HOME ENV=$ENV 'bash -s' << 'ENDSSH'
+ssh -tt bi-${ENV}-ci@${HOST} OOZIE_HOME=$OOZIE_HOME ENV=$ENV INDEX_NAME=$INDEX_NAME 'bash -s' << 'ENDSSH'
```

Comment:
I missed this on the previous review. It looks like youre setting environment variables here. That seems unnecessary? What's the reason?

Reply:
I need to pass variables from the script into `ssh`, I didn't manage to do it by escaping them and this looks far more readable.

Reply:
Ok, I understand why you have done this now, but I don't think these are args, these are actually environment variables (happy to be proven wrong).